### DISCUSSION TITLE
Welcome to the ESPHome Luxtronik V1 discussion forum

### DISCUSSION THREAD

**Discussion by jensrossbach** [jensrossbach/esphome-luxtronik-v1#1] [Announcements]
## 👋 Welcome!
Please use this forum to:
* 🤔 ask questions about ESPHome Luxtronik V1
* 🙏 request help for specific topics
* 💡 propose new features or discuss existing functionality
* 👍 express your opinion about ESPHome Luxtronik V1

Please do **not** create issues for above concerns.

📢 The forum will also be used to announce new versions of ESPHome Luxtronik V1, so stay tuned and have fun! 😀

**Comment by Arni1000** [comment]
Hello,
unfortunately I can't connect to the Luxtronik_V1. 
Flashing the ESP works great. 
All entities are created in the Home Assistant. 
Now my question. Which firmware is required in the Luxtronik_V1? I have version V2.20 installed.
Maybe you can help me.
Here is a section of the log file
[logs_luxtronik-v1-esp32_logs.txt](https://github.com/user-attachments/files/19053534/logs_luxtronik-v1-esp32_logs.txt)

**Reply by jensrossbach** [reply]
I have version 2.33 in my heat pump.

Have you unlocked the serial interface as described [⇒ here (section Luxtronik Configuration)](https://github.com/jensrossbach/esphome-luxtronik-v1?tab=readme-ov-file#luxtronik-configuration)?

**Reply by Arni1000** [reply]
Thanks for the answer. I had switched to "Standard" in the "Diagnosesystem". It is probably due to the older firmware that does not allow the data to be monitored.
Is there a way to test the communication with the interface to see if the interface is sending anything at all?

**Reply by jensrossbach** [reply]
You can use a PC/laptop and connect it via a USB to serial adapter to the Luxtronik heating control unit. With a terminal program like e.g. PuTTY, you can simply send the commands for the datasets (like for instance 1100, 1200, 1300, ...) and should receive the corresponding responses. So you just type the four digits code and press Enter. This way you can test the serial interface of the Luxtronik and you can exclude any problems with the ESP hardware setup. You have to configure PuTTY to use the serial communication parameters as described for the [UART component](https://github.com/jensrossbach/esphome-luxtronik-v1?tab=readme-ov-file#uart-component) and flow control should be turned off.

**Comment by Arni1000** [comment]
I got also a null modem adapter and connected it in between
I have also swapped TX and RX.
